src/edu/washington/escience/myria/accessmethod/JdbcAccessMethod.java

```diff
@@ -425,6 +425,22 @@ public void dropAndRenameTables(final RelationKey oldRelation, final RelationKey
   }
 
   @Override
+  public void dropTableIfExistsCascade(final RelationKey relationKey) throws DbException {
+    switch (jdbcInfo.getDbms()) {
+      case MyriaConstants.STORAGE_SYSTEM_MYSQL:
+        LOGGER.warn("MySQL does not implement CASCADE");
```

Comment:
Okay I just looked at it again (I wasn't sure earlier). For now, I'll just add an ```UnsupportedOperationException```  for this case